content/en/metrics/overview.md

```diff
@@ -0,0 +1,138 @@
+
+##### APM
+
+[Generate custom metrics from your ingested spans][15] to visualize anomalies and trends across any parameters that are important to your business context.
+
+##### Real User Monitoring (RUM)
+
+[Generate custom metrics from RUM events][16] to summarize the data from your RUM events, so you can visualize and alert on the user behaviors that are most impactful to your organization.
+
+##### Processes
+
+[Generate custom process-based metrics][17] to monitor the resource consumption of your processes, as well as any other process-related behavior that might be important to your business needs.
+
+##### Events
+
+[Generate custom event-based metrics][18] for visibility into monitor alerts or any other event-based data ingested by Datadog.
+
+{{% /collapse-content %}} 
+
+### Configurable Processing
+
+The **Configurable Processing** column shows an overview of the configuration options available for your metrics. Click on any of the options for more information and a link to the relevant configuration screen.
+
+{{% collapse-content title="Optimize your custom metrics costs with Metrics without Limits™" level="h4" %}}
+[Metrics without Limits™][19] helps you manage custom metric costs by indexing only the metric tags most valuable to your organization.
+{{% /collapse-content %}} 
+
+{{% collapse-content title="Enable percentiles on distribution metrics" level="h4" %}}
+[Distribution metrics][20] with percentiles enabled show you percentile aggregations of metric values sent from multiple hosts, so you can measure statistical distributions across your entire infrastructure.
+{{% /collapse-content %}} 
+
+{{% collapse-content title="Ingest historical metrics" level="h4" %}}
+[Historical Metrics Ingestion][21] enables you to ingest metric values with timestamps older than one hour from the time of submission.
+{{% /collapse-content %}} 
+
+### Available Metrics
+
+The **Available Metrics** column breaks down your total metric volume by standard and custom metrics over the past month.
+
+## Your metrics by source
+
+This section contains a tree map with a breakdown of your metric sources and their respective volumes.
+
+{{< img src="metrics/overview/metrics_by_source.png" alt="The your metrics by source section of the metrics overview page" >}}
+
+## Generate metrics from any source
+
+Click any of the options to open the page in Datadog where you can create custom metrics for:
+   - [Logs][14]
+   - [Ingested spans][15]
+   - [RUM events][16]
+   - [Processes][17]
+   - [Events][18]
+
+## Metrics available for querying
+
+Use the search bar in this section to view the latest data and configuration options of any of your metrics. You can search all its tags, or click to investigate the metric further in the Metrics Explorer or Summary pages.
```

Comment:
can we provide the in app links for both the explorer and summary pages here

Reply:
Added in [803ca9f](https://github.com/DataDog/documentation/pull/25830/commits/803ca9ff51587e3b1aad595e46e6e064d172caed)